site/content/docs/user/known-issues.md

```diff
@@ -60,7 +60,7 @@ while building a kind cluster:
 
 * Host machine is behind a proxy
 * Usage of Docker version 18.09
-* Building kind in Google Cloud Console
+* Building kind in the Google Cloud Console
```

Comment:
unrelated to this PR, but this should really be "creating" a cluster rather than building, building kind there should work fine, creating a cluster won't quite.